pkg/apis/rollouts/v1alpha1/generated.proto

```diff
@@ -590,8 +590,11 @@ message IstioVirtualService {
   // Name holds the name of the VirtualService
   optional string name = 1;
 
-  // Routes are list of routes within VirtualService to edit. If omitted, VirtualService must have a single route
+  // A list of HTTP routes within VirtualService to edit. If omitted, VirtualService must have a single route of this type.
   repeated string routes = 2;
+
+  // A list of TLS/HTTPS routes within VirtualService to edit. If omitted, VirtualService must have a single route of this type.
+  repeated TLSRoute tlsRoutes = 3;
```

Comment:
why is there tlsRoutes change?

Reply:
Not sure...it happened when I ran codegen

Reply:
Not sure. It changed when I ran codegen.

Reply:
I think previous commits may somehow did not generate it because of un-rebased changes.